contrib/babelfishpg_tsql/sql/sys_functions.sql

```diff
@@ -307,7 +307,7 @@ LANGUAGE SQL IMMUTABLE PARALLEL RESTRICTED;
 CREATE OR REPLACE FUNCTION sys.suser_sid(IN login SYS.SYSNAME, IN Param2 INT DEFAULT NULL)
 RETURNS SYS.VARBINARY(85) AS $$
     SELECT CASE
-    WHEN login = '' 
+    WHEN login = '' COLLATE sys.database_default
```

Comment:
instead of this lets add collate in arg directly. Applies to everywhere.

Reply:
lets avoid that as I discussed